algorithms/src/std_algorithms/impl/Kokkos_RandomAccessIterator.hpp

```diff
@@ -54,17 +54,18 @@ class RandomAccessIterator< ::Kokkos::View<DataType, Args...> > {
   KOKKOS_DEFAULTED_FUNCTION RandomAccessIterator() = default;
 
   explicit KOKKOS_FUNCTION RandomAccessIterator(const view_type view)
```

Comment:
By the way, using `C++20` in `algorithms/unit_tests/TestRandomAccessIterator.cpp`, the following requirement fails:
````c++20
using T1_t = KE::Impl::RandomAccessIterator<static_view_t>;
static_assert(std::random_access_iterator<T1_t>);
```

Reply:
So it seems `Kokkos::Experimental::Impl::are_random_access_iterators` ... is not complete ?

Reply:
I have not tested it with C++20.